libs/components/indicators/src/lib/modules/tokens/fixtures/token.component.fixture.html

```diff
@@ -1 +1,8 @@
-<sky-token [ariaLabel]="ariaLabel"></sky-token>
+<sky-token
+  [ariaLabel]="ariaLabel"
+  [disabled]="disabled"
+  [dismissible]="dismissible"
+  [focusable]="foc
+"
```

Comment:
This seems like a typo.

Reply:
Huh - how did nothing catch that.... fixed.